.github/workflows/promote-cherry-picks.yml

```diff
@@ -0,0 +1,23 @@
+# Promotes the cherry-picked release into all related channels
+# Triggered by circleci job on amphtml
+
+name: Promote Cherry-Picks
+
+on:
+  workflow_dispatch:
+    inputs:
+      amp-version:
+        description: 'AMP version number (13 digits)'
+        required: true
+        default: ''
```

Comment:
Supernit: I think it's better to not list a default value when an input is required
```suggestion
```